### PR TITLE
Improve support for WMS  server 1.1.1

### PR DESCRIPTION
**Pull request by mdastous-bentley**
Improved the support for WMS 1.1.1 servers.

**Three main changes:**
1. In 1.1.1 and prior, 'SRS' must be used instead of 'CRS'. This is applicable for the following 2 cases:
  a. GetMap request parameter
  b. When reading the GetCapabilities response, the JSON contains the 'SRS' key instead of 'CRS'
2. The axis order in 1.1.1 is always x/longitude y/latitude.  In 1.3.0 it depends on the coord. sys. definition, for EPSG:4326, it must be x/latitude y/longitude. 
3. The cursor position parameters passed to the 'GetFeatureInfo' request must be i/j for 1.3.0 and x/y for 1.1.1.

### CODE REVIEW

File: core/frontend/src/test/tile/map/MapLayerImageryProvider.test.ts [hunk 1]
```diff
@@ -135,7 +136,7 @@ describe("WmsMapLayerImageryProvider", () => {
 
   });
 
-  it("should create a GetMap requests URL using the right 'CRS'", async () => {
+  it.only("should create a GetMap requests URL using the right 'CRS'", async () => {
```
Comment (pmconne) [inline core/frontend/src/test/tile/map/MapLayerImageryProvider.test.ts:139]: Remove `.only`.

File: core/frontend/src/test/tile/map/MapLayerImageryProvider.test.ts [hunk 2]
```diff
@@ -186,8 +190,82 @@ describe("WmsMapLayerImageryProvider", () => {
...
+    // check x/y axis is in the right order
+    const p1 = Point2d.create(bboxArray[0], bboxArray[1]);
+    const refPoint1 = Point2d.create(-180, -85.05112878);
+    const p2 = Point2d.create(bboxArray[2], bboxArray[3]);
+    const refPoint2 = Point2d.create(180, 85.05112878);
+    chai.expect(p1.isAlmostEqual(refPoint1)).to.be.true;
+    chai.expect(p2.isAlmostEqual(refPoint2)).to.be.true;
+  });
+
+  it.only("should create a GetMap requests URL in WMS 1.3.0", async () => {
```
Comment (pmconne) [inline core/frontend/src/test/tile/map/MapLayerImageryProvider.test.ts:231]: .only

File: core/frontend/src/test/tile/map/WmsCapabilities.test.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import { WmsCapabilities } from "../../../tile/map/WmsCapabilities";
+
+const mapProxyDatasetNbLayers = 9;
+describe.only("WmsCapabilities", () => {
```
Comment (pmconne) [inline core/frontend/src/test/tile/map/WmsCapabilities.test.ts:10]: .only

File: core/frontend/src/tile/map/WmsCapabilities.ts [hunk 4]
```diff
@@ -82,13 +82,12 @@ export namespace WmsCapability {
     public readonly subLayers = new Array<SubLayer>();
     private static readonly PREFIX_SEPARATOR = ":";
 
-    constructor(_json: any) {
+    constructor(_json: any, capabilities: WmsCapabilities) {
```
Comment (pmconne) [inline core/frontend/src/tile/map/WmsCapabilities.ts:85]: remove leading underscore from `_json`, it indicates an unused argument.

**Review by pmconne — APPROVED** [review]
nitpicks.

**Comment by mdastous-bentley** [comment]
@Mergifyio backport release/3.1.x

**Pull request merged**